sources/core/Stride.Core.Mathematics/Int3.cs

```diff
@@ -189,6 +199,15 @@ public int Length()
             return (int)Math.Sqrt((X * X) + (Y * Y) + (Z * Z));
         }
 
+        /// <summary>
+        /// Calculates the unrounded length of the vector.
+        /// </summary>
+        /// <returns>The length of the vector unrounded.</returns>
+        public float LengthUnrounded() 
```

Comment:
Can you see about coming up a different name here? Like `PreciseLength` maybe?

Reply:
I actually prefer having `Length` first, so either `LengthPrecise` or keeping `LengthUnrounded` is fine too, helps discoverability with intellisense. His name is very explicit in regard to what the function actually does so I actually lean a bit more towards the author's version right now.
Let's see if anyone else can chime in on this.

Reply:
Sure, it truncates the value so you could change its name to `LengthUntruncated` if you want